README.md

```diff
@@ -1,2 +1,90 @@
-# ibm-cloud-env-golang
-IBM Cloud Environment for Go
+# IBM Cloud Environment
+
+The `ibm-cloud-env-golang` module allows to abstract environment variables from various Cloud compute providers, such as, but not limited to, CloudFoundry and Kubernetes, so the application could be environment-agnostic.
```

Comment:
rephrase: The `ibm-cloud-env-golang` module allows for the abstraction of environment variables from various Cloud compute providers, such as, but not limited to, CloudFoundry and Kubernetes, so the application can be environment-agnostic.

Reply:
We're not sure how to rephrase this. This paragraph was taken from the IBM-Cloud-Env repositiory